File: app/src/main/java/com/example/player/a1610aplayerdemo/fragment/Home/present/SelectPresenter.java
package com.example.player.a1610aplayerdemo.fragment.Home.present;

import com.example.player.a1610aplayerdemo.fragment.Home.bean.SelectorBean;

public class SelectPresenter {
    interface ISelectPresener{
        void getSelectorData();
        void DetachView();
    }
    interface ISelectorView{
        void OnGetDataSuccess(SelectorBean selectorBean);
        void onGetDataFail(String e);
    }
}

File: app/src/main/java/com/example/player/a1610aplayerdemo/fragment/Home/present/SelectPresenter.kt
package com.example.player.a1610aplayerdemo.fragment.Home.present

import com.example.player.a1610aplayerdemo.fragment.Home.bean.SelectorBean
import com.example.player.a1610aplayerdemo.view.BaseView

class SelectPresenter {
    interface ISelectPresener{
        fun getSelectorData()
        fun DetachView()
    }
    interface ISelectorView : BaseView {

        fun onGetDataSuccess(bean: SelectorBean)
        fun onGetDataFailed(errorMsg: String)

    }
}